lib/prerequisite.js

```diff
@@ -40,6 +40,18 @@ module.exports = (input, pkg, opts) => {
 			})
 		},
 		{
+			title: 'Check yarn.lock integrity',
+			enabled: () => opts.yarn,
+			task: () => execa.stdout('yarn', ['check', '--integrity'])
+				.catch(err => {
+					if (err.code === 'ENOENT') {
```

Comment:
Error with yarn installed:
```json
{
  "errno": "ENOENT",
  "code": "ENOENT",
  "syscall": "spawn undefined",
  "killed": false,
  "stdout": "yarn check v0.20.3\ninfo \"fsevents@1.1.1\" is an optional dependency and failed compatibility check. Excluding it from installation.\ninfo Visit https://yarnpkg.com/en/docs/cli/check for documentation about this command.\n",
  "stderr": "warning fsevents@1.1.1: The platform \"win32\" is incompatible with this module.\nerror Lockfile does not contain pattern: \"express@*\"\nerror Integrity hashes don't match, expected \"7f3d1555a591e394ae152de23e3926b01b51c6fe5856b2e9f711bdcb780ecd48\" but got \"bb3ee4fe6e326745ee5b805f2d0053153f42cb188035fd73946cfefe4d832902\"\nerror Found 2 errors.\n",
  "failed": true,
  "signal": null,
  "cmd": "yarn check --integrity",
  "timedOut": false
}
```

Without Yarn installed:
```json
{
  "errno": "ENOENT",
  "code": "ENOENT",
  "syscall": "spawn undefined",
  "killed": false,
  "stdout": "",
  "stderr": "'yarn' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n",
  "failed": true,
  "signal": null,
  "cmd": "yarn check --integrity",
  "timedOut": false
}
```

As of Windows 10 x64